sssom/parsers.py

```diff
@@ -140,6 +141,24 @@ def read_obographs_json(
     )
 
 
+def read_snomed_icd10cm_map_tsv(
+    file_path: str,
+    prefix_map: Dict[str, str] = None,
+    meta: Dict[str, str] = None,
+) -> MappingSetDataFrame:
+    """Parse special SNOMED ICD10CM mapping file and translates it into a MappingSetDataFrame.
+
+    :param file_path: The path to the obographs file
```

Comment:
path to icd10 mapping file, see https://www.nlm.nih.gov/research/umls/mapping_projects/snomedct_to_icd10cm.html

Reply:
Actually I wonder if that file format is something more general - i.e. snomed mapping. There are many more such files with other kinds of mappings in them.. maybe rename methods to "snomed_mapping" rather than "snomed_icd10_mapping"..

Reply:
Good find! Done! Also, I determined that this is a 'complex' mapping based on what you linked earlier.